torch/distributed/_composable/fsdp/__init__.py

```diff
@@ -1,2 +1,8 @@
-from ._fsdp_api import CPUOffloadPolicy, MixedPrecisionPolicy, OffloadPolicy
-from .fully_shard import FSDPModule, fully_shard, register_fsdp_forward_method
+from torch.distributed.fsdp import (
+    CPUOffloadPolicy,
+    FSDPModule,
+    fully_shard,
+    MixedPrecisionPolicy,
+    OffloadPolicy,
+    register_fsdp_forward_method,
+)
```

Comment:
Perhaps added a redirect warning here saying `_composable.fsdp` is now moved to `torch.distributed.fsdp`?

Reply:
Good point! I was thinking we can land a follow-up diff first to migrate the internal callsites and then add a warning for OSS. I wanted to avoid showing the warning to some existing internal customers, which might be disconcerting.